testing/test_config.py

```diff
@@ -352,6 +352,25 @@ def test_inifilename(self, tmpdir):
         assert config.inicfg.get('name') == 'value'
         assert config.inicfg.get('should_not_be_set') is None
 
+    @pytest.mark.issue1618
+    def test_consider_plugin(self, testdir):
+        pytest.importorskip('xdist')
+        print (testdir)
+        testdir.makepyfile(conftest="""
+                pytest_plugins = ['plugin']
+            """,
+            plugin="""
+                raise ImportError
+            """,
+            test_foo="""
+                def test():
+                    pass
+            """,
+        )
+        testdir.inline_run(
```

Comment:
This should verify that the test runs OK, so `-p` could effectively disable the problematic plugin.